aspects/aspects.go

```diff
@@ -378,7 +379,7 @@ func (a *Aspect) Set(databag DataBag, request string, value interface{}) error {
 
 			nestedValue, ok = mapVal[part]
 			if !ok {
-				return badRequestErrorFrom(a, "set", request, `cannot key value with unmatched %q key`, part)
+				return badRequestErrorFrom(a, "set", request, `cannot find nested value with unmatched %q key`, part)
```

Comment:
wouldn't "key %q" be a more consistent with other errors and slightly clearer?

Reply:
Good point, will change that